appeals/web/src/server/app/auth/auth.controller.js

```diff
@@ -26,7 +26,13 @@ export async function startMsalAuthentication(request, response) {
 	// The url from which the OpenID Connect flow was triggered by the
 	// application. Ultimately, we will forward the user to this route at the
 	// end of the authentication journey.
-	const postSigninRedirectUri = request.query.redirect_to || '/';
+	let postSigninRedirectUri = request.query.redirect_to || '/';
+	if (
+		postSigninRedirectUri === '/appeals-service' ||
+		postSigninRedirectUri === '/appeals-service/'
+	) {
+		postSigninRedirectUri = '/appeals-service/personal-list';
```

Comment:
I see you defaulted it here, so my previous comment will be redundant...

Reply:
I've removed this logic now